### PR TITLE
Complete output of docker info

### PR DESCRIPTION
**Pull request by zouyee**
**What type of PR is this?**
/kind flake

**What this PR does / why we need it**:
Complete output of docker info

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by zouyee** [comment]
cc @neolit123

**Review by fabriziopandini — APPROVED** [review]
@zouyee thanks!
/approve
I leave final lgtm to other reviewers

File: cmd/kubeadm/app/util/system/docker_validator.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func (d *DockerValidator) Validate(spec SysSpec) (error, error) {
 
 func (d *DockerValidator) unmarshalDockerInfo(b []byte, info *dockerInfo) error {
 	if err := json.Unmarshal(b, &info); err != nil {
-		return errors.Wrap(err, "could not unmarshal the JSON output of 'docker info'")
+		return errors.Wrapf(err, "could not unmarshal the JSON output(%s) of 'docker info'", b)
```
Comment (neolit123) [inline cmd/kubeadm/app/util/system/docker_validator.go]: please change to:
```
"could not unmarshal the JSON output of 'docker info':\n%s"
```

**Pull request merged**